src/assets/img/favicon/README.md

```diff
@@ -0,0 +1,25 @@
+# Your Favicon Package
```

Comment:
@christophdubach Do you think there is any benefit keeping that file? Otherwise, I'd propose to remove it.